charts/k8ssandra/values.yaml

```diff
@@ -537,6 +537,10 @@ monitoring:
     # not have the ServiceMonitor CRD installed on your cluster, set this value
     # to `false`.
     provision_service_monitors: true
+  # -- Deploy the service monitors on a different namespace than the release namespace.
+  # This should only be set if the kube-prometheus stack isn't installed in the release namespace
+  serviceMonitor: {}
```

Comment:
Nit: shouldn't we use `serviceMonitors` in the plural here, since we can have up to 2 service monitors deployed?

Reply:
yup, good point 👍